README.md

```diff
@@ -21,9 +21,9 @@ import v8 "rogchap.com/v8go"
 
 ```go
 ctx := v8.NewContext() // creates a new V8 context with a new Isolate aka VM
-ctx.RunScript("const add = (a, b) => a + b", "math.js") // executes a script on the global context
-ctx.RunScript("const result = add(3, 4)", "main.js") // any functions previously added to the context can be called
-val, _ := ctx.RunScript("result", "value.js") // return a value in JavaScript back to Go
+ctx.CompileAndRun("const add = (a, b) => a + b", "math.js") // executes a script on the global context
+ctx.CompileAndRun("const result = add(3, 4)", "main.js") // any functions previously added to the context can be called
+val, _ := ctx.CompileAndRun("result", "value.js") // return a value in JavaScript back to Go
```

Comment:
@rogchap an alternative here could be `RunUncompiledScript` to differ more clearly from `RunScript`